env-android.yml

```diff
@@ -4,16 +4,17 @@ channels:
 dependencies:
   - ccache=4.10.1
   - python=3.9
-  - nodejs=18.16.*
   - pip=22.3.1
   - rust=1.75
+  - go=1.24.5
   - rust-std-armv7-linux-androideabi=1.75
   - rust-std-x86_64-linux-android=1.75
   - rust-std-i686-linux-android=1.75
   - rust-std-aarch64-linux-android=1.75
-  - cmake=3.26.3
+  - cmake=3.31.6
   - ninja=1.11.0
   - conda-forge::openjdk=17
+  - wget
```

Comment:
Why add `wget`? Generally I recommend `curl` if it's something we are scripting as it generally comes pre-installed.